packages/relay/src/RelayServer.ts

```diff
@@ -239,6 +236,9 @@ export class RelayServer extends EventEmitter {
   }
 
   async validatePaymasterReputation (paymaster: Address, currentBlockNumber: number): Promise<void> {
+    if (this._isTrustedPaymaster(paymaster)) {
```

Comment:
Please add a test for that :)

Reply:
There is no test for `validatePaymasterReputation`, so this addition is to an untested method, therefore testing it is a separate task in my opinion.